pkg/ccl/crosscluster/logical/lww_row_processor.go

```diff
@@ -259,6 +284,41 @@ func (p *failureInjector) injectFailure() error {
 	return nil
 }
 
+func (srp *sqlRowProcessor) HandleBatch(
+	ctx context.Context, batch []streampb.StreamEvent_KV,
+) (batchStats, error) {
+	ctx, sp := tracing.ChildSpan(ctx, "sqlRowProcessor.HandleBatch")
+	defer sp.Finish()
+
+	stats := batchStats{}
+	var err error
+	if len(batch) == 1 {
+		if srp.spec.Discard == jobspb.LogicalReplicationDetails_DiscardAllDeletes && len(batch[0].KeyValue.Value.RawBytes) == 0 {
+			return stats, nil
+		}
+		s, err := srp.ProcessRow(ctx, nil /* txn */, batch[0].KeyValue, batch[0].PrevValue)
+		if err != nil {
+			return stats, err
+		}
+		stats.Add(s)
+	} else {
+		err = srp.db.Txn(ctx, func(ctx context.Context, txn isql.Txn) error {
```

Comment:
this path should error out, right?

Reply:
I think we discussed in slack and this is fine, right?

Reply:
yup, ignore